src/components/organisms/Dashboard/Tableview/InfoTab.tsx

```diff
@@ -0,0 +1,93 @@
+import {useEffect, useState} from 'react';
+
+import {Tag} from 'antd';
+
+import {K8sResource} from '@models/k8sresource';
+
+import {useAppSelector} from '@redux/hooks';
+
+import {timeAgo} from '@utils/timeAgo';
+
+import * as S from './InfoTab.styled';
+
+export const InfoTab = ({resourceId}: {resourceId: string}) => {
+  const resourceMap = useAppSelector(state => state.main.resourceMap);
```

Comment:
Can we directly return from here the resource, without needing an extra useEffect and useState ? 

```
useAppSelector(state => state.main.resourceMap[resourceId]);
```